pwndbg/disasm/__init__.py

```diff
@@ -138,66 +190,93 @@ def get_disassembler(pc):
     )
 
 
-class SimpleInstruction:
-    def __init__(self, address) -> None:
-        self.address = address
-        ins = gdb.newest_frame().architecture().disassemble(address)[0]
-        asm = typing.cast(str, ins["asm"]).split(maxsplit=1)
-        self.mnemonic = asm[0].strip()
-        self.op_str = asm[1].strip() if len(asm) > 1 else ""
-        self.size = ins["length"]
-        self.next = self.address + self.size
-        self.target = self.next
-        self.groups: list[Any] = []
-        self.symbol = None
-        self.condition = False
+# If passed an emulator, this will pass it to the DisassemblyAssistant
+# which will single_step the emulator to determine the operand values before and after the instruction executes
```

Comment:
Should this maybe be a docstring of this function? Note that a docstring can be read via `help(get_one_instruction)` in a debugger or IPython session. This comment can't.